### ISSUE TITLE
Open bookmark in new tab from menu

### ISSUE THREAD

**Issue by Poopooracoocoo** [duckduckgo/Android#690]

**Comment by cmonfortep** [comment]
Hey @Poopooracoocoo, could you add a description to his issue? I would like to understand better how is the proposed user flow. Also, what means "from menu" exactly? We have many screens related to bookmarks or the browser, are you talking about the screens where we show all user active tabs?

**Comment by Poopooracoocoo** [comment]
@cmonfortep sorry. I thought I made this clear. 

I was referring to the bookmark manager and the menu of bookmarks themselves. In Chrome and other browsers you can tap and hold to open it in a new tab. DuckDuckGo has chosen to go with a menu instead and so that's where "Open in new tab" would go.

**Comment by cmonfortep** [comment]
Currently, a bookmark will be opened in a new tab by default. Is there something you are missing?

**Comment by Poopooracoocoo** [comment]
@cmonfortep sorry! i meant to say that it would be nice if tapping opened it in the current tab and if you could open it in a new tab from the menu.

**Comment by sbconslt** [comment]
Came to request this same feature. If I may recommend: this issue would be much clearer if its title was changed to "Open bookmarks in current tab by default". I don't even want to think about how much time I've wasted in the tabs view closing previous tabs after opening bookmarks. As OP said the existing behavior to open in a new tab could be retained as a menu option.

**Comment by cmonfortep** [comment]
Hello @Poopooracoocoo, sorry for the late response here.

After internal discussion with the team, we've agreed that there's no plan in the foreseeable future to support this request. We think our current implementations meets what most of the users will expect, we haven't received more feedback similar to this one. Should this change in the future, we'll update this conversation appropriately.

**Comment by rpet** [comment]
I completely agree with @sbconslt : 

1. This issue would be much clearer if its title was changed to "Open bookmarks in current tab by default" or "Allow bookmarks to be openend in the current tab". I was about to open a duplicate issue when I finally found this one.
2.  Make it configurable in Settings, allowing each user to toggle to their preference. The existing behavior would be the default so users that are happy how things are won't be disturbed.
